src/jit/MemoryInl.h

```diff
@@ -923,6 +991,186 @@ static void emitMemory(sljit_compiler* compiler, Instruction* instr)
     }
 }
 
+#if defined(ENABLE_EXTENDED_FEATURES)
```

Comment:
Do we need this code at this point?

Reply:
I'm not entirely sure which part of it you mean;
The callbacks are used already on 32 bit, and the switches set the parameters for checking the memory and deciding between native load-store / callback

Reply:
The `case ByteCode::I64AtomicLoad8UOpcode:` is handled by emitLoad, and most other as well. This code also has a , `case ByteCode::I64AtomicLoad8UOpcode:` and that is not correct. The only one needs an emulation is 64 bit load/store on 32 bit.